omnibus/config/software/habitat.rb

```diff
@@ -24,23 +24,29 @@
 default_version "1.6.420"
 linux_sha = "2718ac16e8872bf058d47004b1242574f26a49886a9b52e8cbbad3023cd6e314"
 darwin_sha = "1dd86170bbbc3a93ea7a88cb32f2b0e9f1ce30dffc9ef316f25103a7c497a5de"
+darwin_m1_sha = "5ce236f449f2d4bc1134a7946171069ad69a61e761124614dab52de65618ff8f"
 windows_sha = "865f7dc2c079ca8e8d0fbcd8998451ee4e0fa905c10e053809d39d5195039b81"
 # END DO NOT MODIFY
 
 if windows?
-  suffix = "windows.zip"
+  suffix = "x86_64-windows.zip"
   sha256 = windows_sha
 elsif linux?
-  suffix = "linux.tar.gz"
+  suffix = "x86_64-linux.tar.gz"
   sha256 = linux_sha
-elsif mac?
-  suffix = "darwin.zip"
-  sha256 = darwin_sha
+elsif mac? 
+  if /arm64/ =~ RUBY_PLATFORM
```

Comment:
You can probably just use the `arm?` helper here:

https://docs.chef.io/infra_language/checking_architectures/